addons/l10n_OCB/__openerp__.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 OCA (<http://odoo-community.org>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'OCB translations',
+    'version': '0.1',
+    'author': 'OCA',
+    'sequence': 200,
+    'category': 'Localization',
+    'website': 'http://odoo-community.org',
+    'summary': 'Module used to host translations specific to OCB.',
+    'description': "",
```

Comment:
putting a description here, or replacing this key with a README.rst file should get runbot green.